src/contracts/interfaces/IStakeRegistry.sol

```diff
@@ -86,13 +98,24 @@ interface IStakeRegistry is IRegistry {
      */
     function getTotalStakeAtBlockNumberFromIndex(uint8 quorumNumber, uint32 blockNumber, uint256 index) external view returns (uint96);
 
+    /**
+     * @notice Returns the most recent stake weight for the `operatorId` for quorum `quorumNumber`
+     * @dev Function returns weight of **0** in the event that the operator has no stake history
+     */
+    function getCurrentOperatorStakeForQuorum(bytes32 operatorId, uint8 quorumNumber) external view returns (uint96);
+
+    /**
+     * @notice Returns the stake weight from the latest entry in `_totalStakeHistory` for quorum `quorumNumber`.
+     * @dev Will revert if `_totalStakeHistory[quorumNumber]` is empty.
+     */
+    function getCurrentTotalStakeForQuorum(uint8 quorumNumber) external view returns (uint96);
+
     /**
      * @notice Checks that the `operator` was active at the `blockNumber`, using the specified `stakeHistoryIndex` as proof.
-     * @param operator is the operator of interest
+     * @param operatorId is the id of the operator of interest
      * @param blockNumber is the block number of interest
      * @param quorumNumber is the quorum number which the operator had stake in
-     * @param stakeHistoryIndex specifies an index in `operatorIdToStakeHistory[operatorId]`, where `operatorId` is looked up
-     * in `registryCoordinator.getOperatorId(operator)`
+     * @param stakeHistoryIndex specifies an index in `operatorIdToStakeHistory[operatorId]`
```

Comment:
specifically, this is the index of the entry against which the claim is checked, right? makes sense to add wording to this effect.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/0eff79589dccefcbdee149433b9769d122483cfc